mParticle-Apple-SDK/Network/MPURLRequestBuilder.m

```diff
@@ -170,7 +170,7 @@ - (NSMutableURLRequest *)build {
                 kits = nil;
             }
             
-            NSMutableArray<NSNumber *> *configuredKitIds = [[NSMutableArray alloc] initWithArray:[[MParticle sharedInstance].kitContainer configuredKitsRegistry]];
+            NSArray<NSNumber *> *configuredKitIds = MParticle.sharedInstance.kitContainer.configuredKitsRegistry.copy;
```

Comment:
Ahh I didn't realize you didn't actually need a mutable array, and all you're doing is converting it to a string anyway. In that case, there's no need for the copy. This line should just be `NSArray<NSNumber *> *configuredKitIds = MParticle.sharedInstance.kitContainer.configuredKitsRegistry;`

Alternatively, you could remove this line altogether and replace line 175 with `kits = [MParticle.sharedInstance.kitContainer.configuredKitsRegistry componentsJoinedByString:@","];`

Reply:
sweet, gotcha...I didn't realize either until I went back to refactor :)